index.js

```diff
@@ -153,12 +178,13 @@ TransformStreamDecode.prototype._flush = function(done) {
       return done(err);
     }
     if (output) {
-      that.push(output);
+      for (var i = 0; i < output.length; i++)
+        that.push(output[i]);
     }
     done();
-  });
+  }, !this.sync);
 };
 
-function decompressStream() {
-  return new TransformStreamDecode();
+function decompressStream(params) {
+  return new TransformStreamDecode(params);
```

Comment:
It’s being passed to the `Transform` constructor, and it’s good practice in general to pass the first argument of stream constructors to the superclass, so that the constructors are compatible and users can set e.g. the high water mark of a stream to have more control over memory usage.